stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -763,6 +764,13 @@ public extension _TensorComputation {
       let joinStatus = pthread_join(pthread, nil)
       internalConsistencyCheck(joinStatus == 0)
       self.pthread = nil
+    } else {
+      internalConsistencyCheck(
+        _RuntimeConfig.usesSynchronousExecution,
+        """
```

Comment:
Put this `"""` at the end of the last line.

Reply:
Done.

Reply:
Did you move the closing delimiter instead of just moving the starting delimiter?

The closing delimiter should start on a new line, but the starting delimiter can be moved.

Reply:
Yes this works. Thanks!